docs/sources/setup-grafana/configure-grafana/configure-tracing/index.md

```diff
@@ -2,41 +2,88 @@
 aliases:
   - ../../troubleshooting/diagnostics/
   - ../enable-diagnostics/
-description: Learn how to configure tracing so that you can troubleshoot Grafana.
+description: Learn how to configure profiling and tracing so that you can troubleshoot Grafana.
+keywords:
+  - grafana
+  - troubleshooting
+  - documentation
+  - guide
 labels:
   products:
     - enterprise
     - oss
-menuTitle: Configure tracing
-title: Configure tracing to troubleshoot Grafana
+menuTitle: Configure profiling and tracing
+title: Configure profiling and tracing to troubleshoot Grafana
 weight: 200
 ---
 
-# Configure tracing to troubleshoot Grafana
+# Configure profiling and tracing to troubleshoot Grafana
 
 You can set up the `grafana-server` process to enable certain diagnostics when it starts. This can be useful
 when investigating certain performance problems. It's _not_ recommended to have these enabled by default.
 
-## Turn on profiling
+## Turn on profiling and collect profiles
 
 The `grafana-server` can be started with the command-line option `-profile` to enable profiling, `-profile-addr` to override the default HTTP address (`localhost`), and
-`-profile-port` to override the default HTTP port (`6060`) where the `pprof` debugging endpoints are available. For example:
+`-profile-port` to override the default HTTP port (`6060`) where the `pprof` debugging endpoints are available. Further, `-profile-block-rate` controls the fraction of goroutine blocking events that are reported in the blocking profile, default `1` (100%) for backward compatibility reasons, and `-profile-mutex-rate` controls the fraction of mutex contention events that are reported in the mutex profile, default 0 (0%). The higher the fraction the more overhead it adds to normal operations. Running Grafana with profiling enabled and without block and mutex profiling enabled should only add a fraction of overhead and is suitable for [continuous profiling](https://grafana.com/oss/pyroscope/). Adding a small fraction of block and mutex profiling, such as 10-5 (10%-20%) should in general be fine.
```

Comment:
I think it'd be helpful to direct readers to the Go docs for these since the units are slightly inscrutable:

```suggestion
`-profile-port` to override the default HTTP port (`6060`) where the `pprof` debugging endpoints are available. Further, [`-profile-block-rate`](https://pkg.go.dev/runtime#SetBlockProfileRate) controls the fraction of goroutine blocking events that are reported in the blocking profile, default `1` (i.e. track every event) for backward compatibility reasons, and [`-profile-mutex-rate`](https://pkg.go.dev/runtime#SetMutexProfileFraction) controls the fraction of mutex contention events that are reported in the mutex profile, default 0 (0%). The higher the fraction (that is, the smaller this value) the more overhead it adds to normal operations.

Running Grafana with profiling enabled and without block and mutex profiling enabled should only add a fraction of overhead and is suitable for [continuous profiling](https://grafana.com/oss/pyroscope/). Adding a small fraction of block and mutex profiling, such as 10-5 (10%-20%) should in general be fine.
```